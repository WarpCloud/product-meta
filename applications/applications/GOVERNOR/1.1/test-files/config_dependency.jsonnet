{
  application_type: "GOVERNOR",
  application_name: "governor1",
  application_version: "1.1",
  user_config: {data_share: "true"},
}
