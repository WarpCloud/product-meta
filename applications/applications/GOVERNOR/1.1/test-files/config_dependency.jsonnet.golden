{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "governor1",
            "application_type": "GOVERNOR"
         },
         "configs": {
            "notification_cpu_limit": 2,
            "notification_cpu_request": 0.10000000000000001,
            "notification_memory_limit": 4,
            "notification_memory_request": 1
         },
         "dependencies": [
            {
               "moduleName": "txsql",
               "name": "txsql"
            }
         ],
         "instance_settings": { },
         "moduleName": "notification",
         "name": "governor1-notification",
         "version": "1.1"
      },
      {
         "annotations": {
            "application_name": "governor1",
            "application_type": "GOVERNOR"
         },
         "configs": {
            "governor_cpu_limit": 4,
            "governor_cpu_request": 0.10000000000000001,
            "governor_memory_limit": 8,
            "governor_memory_request": 1
         },
         "dependencies": [
            {
               "moduleName": "hdfs",
               "name": "hdfs"
            },
            {
               "moduleName": "hyperbase",
               "name": "hyperbase"
            },
            {
               "moduleName": "kafka",
               "name": "kafka"
            },
            {
               "moduleName": "notification",
               "name": "governor1-notification"
            },
            {
               "moduleName": "elasticsearch",
               "name": "elasticsearch"
            },
            {
               "moduleName": "txsql",
               "name": "txsql"
            }
         ],
         "instance_settings": { },
         "moduleName": "governor",
         "name": "governor1-governor",
         "version": "1.1"
      }
   ],
   "instance_settings": { },
   "version": ""
}
